pwiz_tools/Skyline/Model/Proteome/ProteinAssociation.cs

```diff
@@ -190,7 +193,8 @@ public void UseBackgroundProteome(BackgroundProteome backgroundProteome, Func<Fa
             return proteinAssociations;
         }
 
-        public class ParsimonySettings
+        [XmlRoot("protein_association")]
+        public class ParsimonySettings : IXmlSerializable
         {
             [Track(ignoreDefaultParent:true)]
             public bool GroupProteins { get; set; }

```

Comment:
Ack! This is an immutable object. It cannot have public setters and it must use Change<property>() methods if it allows any direct changes to the object. Otherwise, it should have a constructor that takes its value, and a validation routine that is called during construction and XML serialization, if the values are at all limited in any way, like MinPeptidesPerProtein being at least non-negative.